CHANGELOG.md

```diff
@@ -47,7 +47,7 @@ Changes that are not related to specific components
 
 #### Fixed
 
-- [Component] What bugs/typos are fixed?
+- [Notification] changed includes to use @use notation
 
```

Comment:
There are actually no changes in Notification?

Reply:
true, mayby it should be just `- Changed includes to use @use notation`